core/server/api/v2/utils/serializers/output/utils/mapper.js

```diff
@@ -70,6 +70,12 @@ const mapPost = (model, frame) => {
     return jsonModel;
 };
 
+const forSettings = (attrs) => {
+    url.forSetting(attrs);
+    return attrs;
+};
+
 module.exports.mapPost = mapPost;
 module.exports.mapUser = mapUser;
 module.exports.mapTag = mapTag;
+module.exports.forSettings = forSettings;
```

Comment:
Maybe to keep the same convention similar to above `mapTag`, `mapUser` etc. we could go with `mapSetting`?

Reply:
yeah this was a mistake 👍